cfgov/v1/forms.py

```diff
@@ -297,6 +331,17 @@ def get_query_strings(self):
 
 
 class EventArchiveFilterForm(FilterableListForm):
+
+    def get_page_set(self):
+        return EventFilterablePagesDocumentSearch(
+            prefix=self.filterable_root,
+            topics=self.cleaned_data.get('topics'),
+            categories=self.cleaned_data.get('categories'),
+            authors=self.cleaned_data.get('authors'),
+            to_date=self.cleaned_data.get('to_date'),
+            from_date=self.cleaned_data.get('from_date'),
```

Comment:
Shouldn't this use `start_dt` and `end_dt`?

Reply:
Oddly enough, the form itself passed the data in as `to_date` and `from_date`. I opted to maintain the values here. We specify start_dt and end_dt as part of the search object within the `filter_dates` function on the Event specific search class.